src/lib/services/digital-asset.service.ts

```diff
@@ -120,6 +120,11 @@ async function deployLSP7DigitalAsset(
 ) {
   const controllerAddress = await signer.getAddress();
 
+  const lsp4TokenType =
+    typeof digitalAssetDeploymentOptions.tokenType === 'string'
+      ? LSP4_TOKEN_TYPES[digitalAssetDeploymentOptions.tokenType]
```

Comment:
Are we sure the value of 

 digitalAssetDeploymentOptions.tokenType

Will always be included in that array? Can thus lead to errors? Is the value of that variable verified?

Reply:
> Are we sure the value of
> 
> digitalAssetDeploymentOptions.tokenType
> 
> Will always be included in that array? Can thus lead to errors? Is the value of that variable verified?

I guess yes because of the typing. It must be the type of the key names being either `"TOKEN"`, or `"NFT"` or `"COLLECTION"`

![image](https://github.com/lukso-network/tools-lsp-factory/assets/31145285/a5e5800b-c0b5-4691-b073-1497d416957b)